pkg/yttlibrary/ip.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package yttlibrary
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/k14s/starlark-go/starlark"
+	"github.com/k14s/starlark-go/starlarkstruct"
+	"github.com/k14s/ytt/pkg/orderedmap"
+	"github.com/k14s/ytt/pkg/template/core"
+)
+
+var (
+	IPAPI = starlark.StringDict{
+		"ip": &starlarkstruct.Module{
+			Name: "ip",
+			Members: starlark.StringDict{
+				"parse_addr": starlark.NewBuiltin("ip.parse_addr", core.ErrWrapper(ipModule{}.ParseAddr)),
+				"parse_cidr": starlark.NewBuiltin("ip.parse_cidr", core.ErrWrapper(ipModule{}.ParseCIDR)),
+			},
+		},
+	}
+)
+
+type ipModule struct{}
+
+// AddrValue stores a parsed IP
+type AddrValue struct {
```

Comment:
```suggestion
type IPAddrValue struct {
```

Rationale: the whole slew of types used in the ytt standard library live in the same Go package: `yttlibrary`. Use the type name namespace to make unambiguous (e.g. against a future AddrValue type from another ytt standard library module).

Reply:
fixed